include/flatbuffers/reflection.h

```diff
@@ -95,6 +95,13 @@ inline const Table *GetAnyRoot(const uint8_t *flatbuf) {
   return GetRoot<Table>(flatbuf);
 }
 
+inline Table *GetAnySizePrefixedRoot(uint8_t *flatbuf) {
+  return GetMutableSizePrefixedRoot<Table>(flatbuf);
+}
+inline const Table *GetAnySizePrefixedRoot(const uint8_t *flatbuf) {
```

Comment:
the pointer can be const too. Same for the above method.

Reply:
Not in this overload, which calls (GetMutableSizePrefixedRoot, see GetAnyRoot.